adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -180,13 +189,6 @@ - (BOOL)handlesUserImpressions {
   return YES;
 }
 
-- (GADNativeAdImage *)imageWithUrlString:(NSString *)urlString {
-  NSURL *url = [NSURL URLWithString:urlString];
-  NSData *data = [NSData dataWithContentsOfURL:url];
-  UIImage *image = [UIImage imageWithData:data];
-  return [[GADNativeAdImage alloc] initWithImage:image];
-}
-
```

Comment:
Why is this method removed? Seems like we can use it above?

Reply:
It moved in `- (void)loadRequireData`

Reply:
We can keep this method.

Reply:
This method  was used to load icon image, but now the loading progress is implemented like this:
> GADNativeAdImage *image = [[GADNativeAdImage alloc] initWithImage:[UIImage imageWithData:data]];
> self->_icon = image;

so we don't need this method anymore